src/main/java/io/fabric8/jenkins/openshiftsync/OpenShiftToken.java

```diff
@@ -0,0 +1,37 @@
+package io.fabric8.jenkins.openshiftsync;
+
+import com.cloudbees.plugins.credentials.CredentialsScope;
+import com.cloudbees.plugins.credentials.impl.BaseStandardCredentials;
+import hudson.Extension;
+import hudson.util.Secret;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+// TODO: merge with https://github.com/jenkinsci/kubernetes-plugin/blob/master/src/main/java/org/csanchez/jenkins/plugins/kubernetes/OpenShiftTokenCredentialImpl.java  ?
+public class OpenShiftToken extends BaseStandardCredentials {
+
+    private final Secret secret;
+
+    @DataBoundConstructor
+    public OpenShiftToken(CredentialsScope scope, String id, String description, Secret secret) {
+        super(scope, id, description);
+        this.secret = secret;
+    }
+
+    public String getToken() {
+        return secret.getPlainText();
```

Comment:
if we later want people to be able to store a username/password as their credential object, will this be extensible to support that in a backwards compatible way?

what's the "format" for the Secret?  just free-form text the user supplies?